plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/save-hub.tsx

```diff
@@ -179,8 +183,27 @@ export const SaveHub = () => {
 			return (
 				<Button
 					variant="primary"
+					isBusy={ isResolving }
 					onClick={ () => {
-						sendEvent( 'FINISH_CUSTOMIZATION' );
+						const homeUrl: string = getSetting( 'homeUrl', '' );
+						// Wait for 5 seconds before redirecting to the transitional page. This is to ensure that the site preview image is refreshed.
+						const WAIT_BEFORE_REDIRECT = 5000;
```

Comment:
Is there a better way to detect when mShot is finished? Maybe its better addressed in a follow up. At the least a spinner on the "Done" button would be good.

Reply:
AFAIK, there's no way to detect that. Added a spinner in 58a5546fdee359b600b4da40c2b8c081342634b1.